src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int ARRAY_LENGTH_MULTIPLIER = 2;
+    private Node<K, V>[] table;
+    private int threshold;
```

Comment:
I'd recommend you to count threshold where you need instead of save it as additional class variable